examples/mark_safe_insecure.py

```diff
@@ -14,6 +14,14 @@ def insecure_function(text, cls=''):
 safestring.SafeBytes(my_insecure_str)
 
 
+text, url = choice(['a', 'b'])
```

Comment:
Why is this being added?  I locally took this changes in hopes of reproducing the bug, the test still pass.  Please add a unit test that recreates the error described in the bug.

Reply:
@ericwb the issue here is when the rule tries to detect the type of the variables used in the call "mark_safe".
I juat checked it and without that fix the error continues happens